include/sound/soc.h

```diff
@@ -1232,9 +1232,6 @@ struct snd_soc_pcm_runtime {
 	/* bit field */
 	unsigned int dev_registered:1;
 	unsigned int pop_wait:1;
-
-	/* private data - core does not touch */
-	void *private;
```

Comment:
Fwiw, I can't remember if I added this for SOF or pre SOF when developing original ASoC. It may need a seperate patch for upstreaming.